README.md

```diff
@@ -14,6 +14,9 @@ running the responsive public website and set `development` to `true`
 to always return `true` in public_website or responsive_site
 4. Run `bundle exec jekyll serve --watch`
 
+The test harness uses a key pointing to one of the partner tools. 
+In order to test a partner tool in the test environment, set the key to point to the relevant partner tool here: https://github.com/moneyadviceservice/test-harness/blob/gh-pages/_config.yml#L4
```

Comment:
Hey @tomas-stefano see below for some suggestions on how to get the message across better. It will be helpful to see the key(s) to be changed before navigating to the file.

In order to test a partner tool in the test environment, set the key to point to the relevant partner tool by changing the value of the `qa_partner_tools_url` here: https://github.com/moneyadviceservice/test-harness/blob/gh-pages/_config.yml#L4. The default value is set to `https://preview-partner-tools.dev.mas.local`. 

You may choose to make changes if you are using an alternative environment.

```
# in file: gh-pages/_config.yml
qa_partner_tools_url: 'https://preview-partner-tools.dev.mas.local'
```